nrf70_bm_lib/docs/source/nrf70_bm_lib.rst

```diff
@@ -122,7 +122,7 @@ Optimizing scan operation
 The nRF70 Series BM library provides a single API to perform a Wi-Fi scan operation.
 The scan operation is optimized to provide a wide range of scan configuration parameters.
 
-Please see `Optimizing scan operation <https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/protocols/wifi/scan_mode/scan_operation.html>`_ for more information.
+Please see `Optimizing scan operation <https://docs.nordicsemi.com/bundle/ncs-2.7.0/page/nrf/protocols/wifi/scan_mode/scan_operation.html>`_ for more information.
```

Comment:
@krish2718 is this because the information is not part of the latest docs? (Or has moved?)